i3pystatus/core/util.py

```diff
@@ -373,7 +373,7 @@ def __new__(cls):
         try:
             socket.create_connection(cls.address, 1).close()
             return True
-        except OSError:
+        except (OSError, socket.gaierror):
```

Comment:
I'm using python3.5 and the original code isn't catching gaierror. Is it possible that this was later reverted?